uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,63 @@
+
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Removes duplicated "Localise CiviHR" menu item
+   */
+  public function upgrade_1020() {
+    $this->up1020_deleteLocaliseCiviHRMenuItem();
+    $this->up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist();
+
+    return TRUE;
+  }
+
+  private function up1020_deleteLocaliseCiviHRMenuItem() {
```

Comment:
For me the method name is explanation enough, but I got the impression before that we try to put an explanatory docblock on all methods, what do you think?

Reply:
I agree, but since we've been adding docblocks to all methods, I'll do that here too